core/node/vm_runner/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "zksync_vm_runner"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+multivm.workspace = true
+vise.workspace = true
+zksync_types.workspace = true
+zksync_dal.workspace = true
+zksync_config.workspace = true
```

Comment:
`zksync_config` is redundant here

Reply:
Huh, funny that [udeps](https://github.com/est31/cargo-udeps) didn't discover that.
Good catch!

Reply:
@popzxc  same for other `Cargo.toml`s(for example `core/node/db_pruner/Cargo.toml`)
please check them all

Reply:
Of course